nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/settings/ui/registry-clients/registry-clients.component.spec.ts

```diff
@@ -228,10 +229,37 @@ describe('RegistryClients', () => {
                         uri: mockRegistryClientEntity.uri,
                         fromTimestamp: '2023-10-08T12:00:00.000Z',
                         componentId: mockRegistryClientEntity.id,
-                        componentType: ComponentType.FlowRegistryClient
+                    componentType: ComponentType.FlowRegistryClient
+                }
+            })
+        );
+    });
+
+    it('should dispatch navigateToComponentDocumentation action when viewRegistryClientDocumentation is called', async () => {
+        const { component, store } = await setup();
+        const mockRegistryClientEntity = createMockRegistryClientEntity();
+        jest.spyOn(store, 'dispatch');
+
+        component.viewRegistryClientDocumentation(mockRegistryClientEntity);
+
+        expect(store.dispatch).toHaveBeenCalledWith(
+            navigateToComponentDocumentation({
+                request: {
+                    backNavigation: {
+                        route: ['/settings', 'registry-clients', mockRegistryClientEntity.id],
+                        routeBoundary: ['/documentation'],
+                        context: 'Registry Client'
+                    },
+                    parameters: {
+                        componentType: ComponentType.FlowRegistryClient,
+                        type: mockRegistryClientEntity.component.type,
+                        group: mockRegistryClientEntity.component.bundle.group,
+                        artifact: mockRegistryClientEntity.component.bundle.artifact,
+                        version: mockRegistryClientEntity.component.bundle.version
                     }
-                })
-            );
-        });
+                }
+            })
+        );
     });
 });
+});
```

Comment:
I _think_ there is an extra `});` here. Can you please remove and it run prettier to format everything? Run the following from `/nifi/nifi-frontend/src/main/frontend`:

`npx nx prettier:format nifi`